src/Data/Rational/Unnormalised/Properties.agda

```diff
@@ -946,6 +1010,33 @@ neg-distribˡ-* p q = *≡* $ cong (ℤ._* (↧ p ℤ.* ↧ q))
 neg-distribʳ-* : ∀ p q → - (p * q) ≃ p * - q
 neg-distribʳ-* p q = *≡* $ cong (ℤ._* (↧ p ℤ.* ↧ q))
                          $ ℤ.neg-distribʳ-* (↥ p) (↥ q)
+
+------------------------------------------------------------------------
+-- Properties of _*_ and _/_
+
+*-homoˡ-id : ∀ p {q r} pr≢0 r≢0 → ((ℤ.+ p ℤ.* q) / (p ℕ.* r)) {pr≢0} ≃ (q / r) {r≢0}
```

Comment:
Variable names `p`, `q` and `r` should be used for rationals only. Use `m`, `n` for naturals and `i` and `j` for integers.

Reply:
Not fixed?